DiceKit/Die.swift

```diff
@@ -116,3 +116,14 @@ extension Die: ExpressionType {
 public func == (lhs: Die, rhs: Die) -> Bool {
     return lhs.sides == rhs.sides
 }
+
+// MARK: - Debug
+extension Die: CustomDebugStringConvertible {
+    
+    public var debugDescription: String {
+        get {
+            return "I am a Die! I have \(self.sides) sides!"
```

Comment:
What we should show is whatever you would expect to show on the right hand side in a playground. Let's see what some defaults are. `po` in the debugger gives a nice looking view

```
(lldb) po Die(sides: 6)
▿ DiceKit.Die
  - sides : 6
```

And this is default `debugPrint`:
```
(lldb) po debugPrint(Die())
DiceKit.Die(sides: 6)
```

I don't know about you, but `DiceKit.Die(sides: 6)` is very verbose. I think something like `Die(6)` might be better debugPrint.

Where it will get really interesting is with expressions. Those we will want to basically reverse into operations.